src/Modules/CalcPerform.lua

```diff
@@ -3141,10 +3124,40 @@ function calcs.perform(env, avoidCache)
 					hexCastRate = hexCastRate - m_ceil(hexCastRate - maxVixenTriggerRate)
 				end
 			end
+
+			-- Doom Blast stores three uses. If we assume that the spell is triggered again before all three
+			-- charges have been restored, we can ignore the rate cap imposed on other skills (there are no breakpoints).
+			-- That is why we have separated this piece of code rather than calling getTriggerActionTriggerRate().
+			local baseActionCooldown = env.player.mainSkill.skillData.cooldown
+			local icdr = calcLib.mod(env.player.mainSkill.skillModList, env.player.mainSkill.skillCfg, "CooldownRecovery")
+			local cooldownOverride = env.player.mainSkill.skillModList:Override(env.player.mainSkill.skillCfg, "CooldownRecovery")
```

Comment:
Small nitpick but as this code is now specific to this block and i doubt Doom Blast will ever get a cool down override mechanic this and the cooldown override breakdown below could be removed to slim the code down.